packages/plugin-commands-script-runners/src/run.ts

```diff
@@ -107,7 +107,7 @@ For options that may be used with `-r`, see "pnpm help recursive"',
       FILTERING,
     ],
     url: docsUrl('run'),
-    usages: ['pnpm run <command> [-- <args>...]'],
+    usages: ['pnpm run <command> [--] [<args>...]'],
```

Comment:
Should we just drop the `[--]` part here like `pnpm --help exec` does?

Reply:
drop it